parm/config/config.base.emc.dyn

```diff
@@ -12,19 +12,9 @@ export machine="@MACHINE@"
 export RUN_ENVIR="emc"
 
 # Account, queue, etc.
-if [ $machine = "HERA" ]; then
-
-    export ACCOUNT="fv3-cpu"
-    export QUEUE="batch"
-    export QUEUE_ARCH="service"
-
-elif [ $machine = "WCOSS_C" -o $machine = "WCOSS_DELL_P3" ]; then
-
-    export ACCOUNT="GFS-DEV"
-    export QUEUE="dev"
-    export QUEUE_ARCH="dev_transfer"
-
-fi
+export ACCOUNT="@ACCOUNT@"
```

Comment:
Kate, removing the references to specific machines in config.base is fine, but where will these parameters be defined, including account, queue and directory settings ?  Do  you have a script that defines these parameters for each machine and then call the python setup scripts ?  No one can remember all the parameters for different platforms.  I have been using a script, for example, /gpfs/dell6/emc/modeling/noscrub/emc.glopara/para_gfs/misc/rocoto_cycled_warm.sh, to set up my runs.